src/components/application_manager/src/commands/mobile/unsubscribe_button_request.cc

```diff
@@ -60,22 +60,15 @@ void UnsubscribeButtonRequest::Run() {
   const uint32_t btn_id =
       (*message_)[str::msg_params][str::button_name].asUInt();
 
-  if (!app->IsSubscribedToButton(
+  if (app->UnsubscribeFromButton(
```

Comment:
@LevchenkoS , please invert logic, looks confusing , when we passed all checks and send IGNORED